### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by alfredonodo**

### CODE REVIEW

File: apps/client/src/locales/messages.es.xlf [hunk 1]
```diff
@@ -0,0 +1,2651 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+  <file source-language="en" datatype="plaintext" original="ng2.template">
+    <body>
+      <trans-unit id="e19fcf996343543e13789f17b550ab0c08124b5c" datatype="html">
+        <source>Create Account</source>
+		<target state="translated">Crear una cuenta</target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.es.xlf:7]: Can you please replace the tabs with double spaces that it matches the style?

File: apps/client/src/locales/messages.it.xlf [hunk 2]
```diff
@@ -4,6 +4,7 @@
     <body>
       <trans-unit id="e19fcf996343543e13789f17b550ab0c08124b5c" datatype="html">
         <source>Create Account</source>
+		<target state="translated">Crea un account</target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.it.xlf:7]: Here also (tab vs. spaces), maybe the conflicts go away with that change...

**Review by dtslvr — CHANGES_REQUESTED** [review]
Thank you for your team work @casitu and @alfredonodo 🙌🏻

**Comment by dtslvr** [comment]
Related to #1293.

File: apps/client/src/locales/messages.es.xlf (hunk 1 above)
Comment (alfredonodo) [reply apps/client/src/locales/messages.es.xlf]: I did it, but I am not sure if it is better.

File: apps/client/src/locales/messages.it.xlf (hunk 2 above)
Comment (alfredonodo) [reply apps/client/src/locales/messages.it.xlf]: Done

File: apps/client/src/locales/messages.es.xlf (hunk 1 above)
Comment (dtslvr) [reply apps/client/src/locales/messages.es.xlf]: Sorry to bother you, it is still not correct:

messages-xlf

You can see it in [Files changed](https://github.com/ghostfolio/ghostfolio/pull/1292/files): Click _Load diff_

Comment (alfredonodo) [reply apps/client/src/locales/messages.es.xlf]: I am not sure what level of indentation is desired. I took the <source> level and added two spaces.

**Pull request closed without merging**